.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+**JIRA Ticket**: (link)
+
+* Other Relevant Links (Google Groups discussion, related pull requests, Release pull requests, etc.)
+
+# What does this Pull Request do?
+
+A brief description of what the intended result of the PR will be and/or what problem it solves.
+
+# What's new?
+A in-depth description of the changes made by this PR. Technical details and possible side effects.
+
+Example:
+* Changes x feature to such that y
+* Added x
+* Removed y
+
+# How should this be tested?
+
+A description of what steps someone could take to:
+* Reproduce the problem you are fixing (if applicable)
+* Test that the Pull Request does what is intended.
+* Please be as detailed as possible.
+* Good testing instructions help get your PR completed faster.
+
+
+# Additional Notes:
+Any additional information that you think would be helpful when reviewing this PR.
+
+Example:
+* Does this change require documentation to be updated? 
+* Does this change add any new dependencies? 
+* Does this change require any other modifications to be made to the repository (ie. Regeneration activity, etc.)? 
+* Could this change impact execution of existing code?
+
+# Interest parties
```

Comment:
Interest_ed_ parties? (Unless you mean this is more fun than an Interest Group!)